src/quo2/screens/messages/resources.cljs

```diff
@@ -0,0 +1,5 @@
YW0tTMYnDR52KeHhgbhnvZieP1GSt6WfSbKxaDFXiXwzs7MspRIYV07EOJ9xIIXFwXHt9n6wuw8nEqnOB4XESfVbUiZbVPptfS/KtsqZ4RC/zvFnNDFuYQj5vX6XnY+4GUnRXfvHs4y0+4Qt3CQnVn06atUXdOxWezZsw+33PLbkuKePfcCxCRpYKCPvZcP4WilnniIGh9aAJIuZnQ0IMWxGJGGGpmamNjkOAAVR2KWlpPoIu110swQLFBhEHXdkJY2K4alFcP4+CVOnMpy95vyQKRK9rYqj9/lMmuYuHABebI9kQHVZqV1LpnCrkkpXOesUbyvROGV4u8mqWsWluN3PPjgg7FfGAAZhC3rj5VLlRM1QrSTbKpGR8frC+HjH/0ktcE5jJ16Xt68j21QBGNoaAiJRBIh7mwn7a0V0uqFpYRkZ8KqkjY0HWGNtLWbAGhx8hubW+LvcJrL1C+wNCoEQhcxQRy8CvM+fX0RskOT/d9LwlVgwBoSA8Sk9+z4eQ468pL6XnPNNdJjFK6vKLEsF1rMccTG7KuL0b4p5odleY5otK83Fpu+dM/PrlfDK1y/+Yk/eFRtaodr9YohTMRqsY6zlLizpKQB8nSTVq2PiB2LxbBly2bW+gJ7v491mJfv7+3rZuZwIsudbjA9hfsrhImgtuKQpVCO4rDjGktlO+Uyd4YZpbAMuJB8BmtrGQqtPHe2RJJGLjFPy47WvMHSERzERY7SSws/RH0yP0UQbHKTKOfdLqN1WpRSuUHkF7POfpKhIWcg29c+cNVYbCL7qgJw711fzP7O732yVm8039lDbp5jSs0tLXPXI+ijDydOiizHF/lQEcTjCxxgcoZHe1oel+WHi3q3SFpcLtGDFTnrExMrl9E6y+OgyyQMEYU7KpwbD1lbhSAqjs/XGVwhaISfJbpCgJJ4lQTMSznb3hbE0tKS7OvdfJZuGh3veNd/wsEbbqTvsAEROtsWd94lzlKJcTw/58DIRrQ5nZ/+6uMPPfhKa/2lByXHJ2ePTb0wdmQtvUrXxyVrfIa7vn//fhw//iiuvvpqPPLIw8SO3ZicnJQILQ5B1kwxx/NKTSFPkYkTpuWSPG7jdvlkJiiUhuJ3VIKYCGQmm5H9m4NaiR0OvXUKRPCNSp2S2O/m52wjv6hijB7BjTdcJyfRraPTQmFJLkhpXSAnuYh//sEPceInP0ZAddzx9488e/svWuMvPSq7dePw0e9/7/6Q7XR8VAw4L50/g40bt+LMmdNsZcM4f/4cgXCQ6bomW5U4TSptLrqxpUqeCGzAQ/Quso6F4KmUmB20tn308UXN5ht5mc6JlXkprMTE2JSHLVV5wEFkisgg8e+lpTgxhnPAwWG5a1XiEC73+EszFymZL8rg5bIFiU25lWUM9a2794vf/Kfbf9kaf+XfC7zv/e89HPJ6xuYXY/yXkwCUYIoSxZdmJWcfoQuzkkhQcfUhvZaQhxXiK0l6JJzlk+gEwpTTHh8XmZQB8nAnBc/w0Z0RmbC4kMSli0vUHBbii0nEl1I0U5PEmzmyzAUU80VpyggOIOaKwqStiyELKXdr3AW0UwKLcjs3/hz+4Tv34p8fvI/mrvYCF3/4V61Pw6u4vvXd794V7R8Z3n9g365nnnkKV+69Qi7+Yx+/Fcc5CPF7Qmyb4pRoU8rnUDDEmieH546KU1ouEhd/0C1tcpHq4sywTunsNgSlDjFLDA5TcmR4wzL9XRRjfsrdNuJPmH6j6EImAbVAkO3t7eFCx7B1yyg5R5ssMWGPrVs/imuvuR6Hbv4g3++79447/s87X83aXtOfzR06dNPte3ddd1uozUukZlprDdrfa9i8eRO7xAS279jM+puXh6dz2RzrPSB7dKSbhIi6olSsytQVvCAYEkfgBIEyW/VOABTjrzK7h+D1NVNMlhQMk/Y2mpqk4nV2lK1btnFiPYf1I8NYiK8gQ/2xQkYZXxbW3SxW4/E7Tz115lX/5chr/rvBRx998Oi3v/Wj2/bu2RM6e+YMnaAtuPeer+GDH76FpCTH8qA3R09/gMbpxfMz0r62yQmilMyiztfWcpibm5WnxYdH+mhuUCs3RVuElMLtXX2yhZrsHDn29HV834WJS5JZ2kqD2XcV1vO1P/vCZzE+PcVME6JNYkFWtZp3XHzizLHXsp7X9YeTs7Onow//5Jnj09Pz0T5aVMIcdXnI+jhOUml9lakSRbsTh6qrZpEpKv7ExiPnAYIZrmVTMgucBFYfVWCNLTPPWhfnkDi+wOZNm4kf7ZJXzM5M0r2JSMMkn+e4nAC6/8qd+MI3v4oTJ5+inyCXcKJpO26NnXg6htd4va6/Ghse3h373f/6ieGGVb31udOnYiECXZpDEI2LF7JV/G2Pi+VhOHWJ/pk0GVmFxgUzxGVQE4Tb5eBDmCMlskCdMrk9SDlM2030/xl6Bs/RuooxpZswZBcR531VcRi3WpE84cDOPTS8kbUV+1PTx09d93oW/7oD8OL1l3/5/+659tp91/3o/vvvNAx3NplcgmhNbrZCMf9zOBrsCgqBzCcNkFK5KDHBQZ0ghIrQ6XVxhk8MWRrCFdLkmWNd2GCU3JGudimJG6ZGTcDJjjiPKAwPRc1edcWVdxRya8Mzjzz/mlL+Z6837G+Hxd/tfe++rx6sFFy3zc5OR+tMfR+1vuEKkeHVZEkUOUsQx+39bIUuhxdploKw3USQYDtax3FJkASFNWjFC/NTnATPZSsSAJv1RtZw23d2dPQeu/XWW7N4A6435Y+n/+S//cHBkfVbDucq5rWKrUQrVaHesiiU00R6P8lTlHjAAcWMGLikOc0J0siMUkU6kEzF2fdXGZiSHLXpTkesXlN/QCf3vk996g9P4A2+3pQA/Mfr6NHD0T379uw6sP/6XfH5ws5HHn44uhJfCA1vjEbDbX0kTYtyqpNIxGMkM9krrnxLbN+VV81t2tA9xrefUBR3DG/i9W9VHcSXe/LRagAAAABJRU5ErkJggg=="
```

Comment:
Please use images files, this way we probably will have duplicates

Reply:
Done